pkg/cloud/vsphere/services/kubeadm/noderegistration.go

```diff
@@ -27,12 +27,11 @@ import (
 type NodeRegistrationOption func(*kubeadmv1beta1.NodeRegistrationOptions)
 
 // NewNodeRegistration will create a kubeadmNodeRegistrationOptions with some defaults and the ability to provide customizations.
-func NewNodeRegistration(opts ...NodeRegistrationOption) kubeadmv1beta1.NodeRegistrationOptions {
-	nro := &kubeadmv1beta1.NodeRegistrationOptions{}
+func SetNodeRegistration(base *kubeadmv1beta1.NodeRegistrationOptions, opts ...NodeRegistrationOption) kubeadmv1beta1.NodeRegistrationOptions {
```

Comment:
Please lint. It would tell you that the function’s name doesn’t match the comment.

Also, please indicate why the function was renamed?

Reply:
We updated this locally, this is stale branch, git issues

Reply:
Will lint.
I renamed it to match the naming in CAPA.

Reply:
Cool.